nixos/modules/misc/meta.nix

```
@@ -39,6 +39,64 @@ let
     #   { file = "module location"; value = <path/to/doc.xml>; }
     merge = loc: defs: defs;
   };
+
+  /**
+    Custom type for `meta.tests` option.
+  */
+  testsType = lib.mkOptionType {
+    name = "tests";
+    check = value: lib.isFunction value;
```

Was this intentional?

```suggestion
    check = lib.isFunction;
```